enterprise/models.py

```diff
@@ -349,6 +349,11 @@ class Meta:
         help_text=_("Specifies whether the learner should be able to login through enterprise's slug login")
     )
 
+    enable_2U_executive_education_fulfillment = models.BooleanField(
+        default=False,
+        help_text=_("Specifies whether the enterprise should have access to 2U executive education content.")
```

Comment:
nit based on:

> Based upon the internal conversations, the need to distinguish between the unpaid and paid variants/mode of Executive Education courses, and distinguishing 2U’s ExEd courses' enrollment track from the existing Executive Education enrollment type, a new enrollment mode/type is needed across LMS, Discovery, and Ecommerce. That new variant can be termed as **Executive Education (2U)**

Maybe we say "Executive Education (2U)" in the help text and `enable_executive_education_2U_fulfillment` in the model field name to be consistent with this new variant described in the [Exec Ed docs](https://2u-internal.atlassian.net/wiki/spaces/IM/pages/18974245/Executive+Education+Course+Type).